### PR TITLE
[msbuild] Adding W0111 Translations

### PR DESCRIPTION
**Pull request by tj-devel709**

### CODE REVIEW

**Review by whitneyschmidt — APPROVED** [review]

File: tests/msbuild/Xamarin.MacDev.Tasks.Tests/TaskTests/LocalizationIgnore/common-Translations.ignore
Comment (spouliot) [inline tests/msbuild/Xamarin.MacDev.Tasks.Tests/TaskTests/LocalizationIgnore/common-Translations.ignore:2]: ❤️

**Review by spouliot — APPROVED** [review]

**Review by mandel-macaque — APPROVED** [review]
Looks ok, but we there is a file missing to be updated: MSBStrings.resx:
```
 error : 'xlf/MSBStrings.cs.xlf' is out-of-date with 'MSBStrings.resx'. Run `msbuild /t:UpdateXlf` to update .xlf files or set UpdateXlfOnBuild=true to update them on every build, but note that it is strongly discouraged to set UpdateXlfOnBuild=true in official/CI build environments as they should not modify source code during the build. [/Users/builder/jenkins/workspace/xamarin-macios-pr-builder/msbuild/Xamarin.Localization.MSBuild/Xamarin.Localization.MSBuild.csproj]
```

**Review by rolfbjarne — APPROVED** [review]

**Review by dalexsoto — APPROVED** [review]

**Comment by tj-devel709** [comment]
> Looks ok, but we there is a file missing to be updated: MSBStrings.resx:
> 
> ```
>  error : 'xlf/MSBStrings.cs.xlf' is out-of-date with 'MSBStrings.resx'. Run `msbuild /t:UpdateXlf` to update .xlf files or set UpdateXlfOnBuild=true to update them on every build, but note that it is strongly discouraged to set UpdateXlfOnBuild=true in official/CI build environments as they should not modify source code during the build. [/Users/builder/jenkins/workspace/xamarin-macios-pr-builder/msbuild/Xamarin.Localization.MSBuild/Xamarin.Localization.MSBuild.csproj]
> ```

That's strange. When I run 'msbuild /t:UpdateXlf', it says everything is updated?

**Comment by rolfbjarne** [comment]
Something is missing, because if I run `msbuild /t:UpdateXlf` in the msbuild/Xamarin.Localization.MSBuild directory, these files are changed: https://gist.github.com/rolfbjarne/c6f80caa08387ad0ca688caf0b141aeb

**Comment by tj-devel709** [comment]
build

**Comment by tj-devel709** [comment]
@rolfbjarne I saw you mentioned the sims failing in [this](https://github.com/xamarin/xamarin-macios/pull/10038) PR, is this okay to merge due to https://github.com/xamarin/xamarin-macios/issues/10002

**Comment by rolfbjarne** [comment]
@tj-devel709 yes, these test failures are known failures. However, you need @emaf's approval before you can merge.

**Review by emaf — APPROVED** [review]

**Pull request merged**
